### PR TITLE
[bugfix] Read data from blockchain

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- upgrade `@ainblockchain/ain-js` to version 1.10.1.
- add `is_shallow` option when retrieving value.
- fix query apis to read data from blockchain. (in assistant, thread, message)

Related issues:
- https://github.com/ainft-team/ainft-js/issues/284

### CODE REVIEW

File: src/ainft.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export default class AinftJs {
     this._blockchainUrl = blockchainUrl || AIN_BLOCKCHAIN_ENDPOINT[stage];
     this._chainId = chainId || AIN_BLOCKCHAIN_CHAIN_ID[stage];
 
-    this.ain = new Ain(this._blockchainUrl, undefined, this._chainId);
+    this.ain = new Ain(this._blockchainUrl, null, this._chainId);
```
Comment (platfowner) [inline src/ainft.ts:70]: 👍

File: src/utils/validator.ts [hunk 2]
```diff
@@ -6,15 +6,16 @@ import { valueExists, getValue } from './util';
...
+  const objectOwnerPath = `apps/${appId}/owner`;
+  const objectOwner = await getValue(ain, objectOwnerPath);
+  return address === objectOwner;
 };
 
 export const validateObject = async (ain: Ain, objectId: string) => {
   const appId = AinftObject.getAppId(objectId);
   const objectPath = Path.app(appId).value();
-  if (!(await valueExists(ain, objectPath))) {
+  const object = await getValue(ain, objectPath, { is_shallow: true });
```
Comment (platfowner) [inline src/utils/validator.ts:17]: 👍

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by jiyoung-an** [comment]
Thank you for review @platfowner! now merging...

**Pull request merged**